lib/rollbax/item.ex

```diff
@@ -33,4 +38,21 @@ defmodule Rollbax.Item do
       Map.put(data, "custom", meta)
     end
   end
+
+  defp language() do
+    "Elixir (" <> System.version <> ")"
+  end
+
+  defp platform() do
+    :erlang.system_info(:system_version)
+    |> to_string
```

Comment:
There is Atom, which have `String.Chars` protocol. Make sense to use `Atom.to_string/`1 instead?